snirf_specification.md

```diff
@@ -537,6 +537,119 @@ As described below, optional variables `probe.sourceLabels` and
 `probe.detectorLabels` are provided for indicating the instrument specific 
 label for sources and detectors.
 
+#### /nirs(i)/data(j)/measurementLists
+* **Presence**: optional
+* **Type**:  group
+* **Location**: `/nirs(i)/data(j)/measurementLists`
+
+The group for measurement list variables which map the data array onto the probe geometry (sources and detectors), data type, and wavelength. This group's datasets are arrays with size `<number of channels>`, with each position describing the corresponding column in the data matrix. (i.e. the values at `measurementLists/sourceIndex(3)` and `measurementLists/detectorIndex(3)` correspond to `dataTimeSeries(:,3)`).
+
+This group is required only if the indexed-group format `/nirs(i)/data(j)/measurementList(k)` is not used to encode the measurement list. `measurementLists` is an alternative that may offer better performance for larger probes.
+
+The arrays of `measurementLists` are:
+
+#### /nirs(i)/data(j)/measurementLists/sourceIndex 
+* **Presence**: required if measurementLists is present
+* **Type**:  integer 1-D array
+* **Location**: `/nirs(i)/data(j)/measurementLists/sourceIndex`
+
+Must be 1-D array with length equal to the size of the second dimension of `/nirs(i)/data(j)/dataTimeSeries`. Source indices for each channel.
```

Comment:
For this, and all subfields...

The rest of the specification uses a style closer to, e.g., 

"Source indices for each channel. A 1-D array with length equal to the size of the second dimension of ..."

That is to say that generally he description comes first, then additional information such as dimensionality follows, and it is written as a declaration more than an imperative.